### PR TITLE
Include etag in `meta` section of each entities `data`

### PR DESCRIPTION
**Pull request by carlos-alberto**

### CODE REVIEW

File: test/methods/test_patch.py [hunk 1]
```diff
@@ -34,7 +36,7 @@ def test_patch_updates_only_specified_field(flask_client):
     _id = create_response.json['data']['id']
     data = {
         'data': {
-            'type': 'user', 'id': _id, 'attributes': {'name': 'Carly'}
+            'type': 'user', 'id': _id, 'attributes': {'name': 'Carly'},
```
Comment (siame) [inline test/methods/test_patch.py]: Why the comma here?

Comment (carlos-alberto) [inline test/methods/test_patch.py]: No reason, will fix.

**Comment by siame** [comment]
📦

**Comment by villaaston** [comment]
🍧

**Pull request merged**